rasterio/rio/features.py

```diff
@@ -192,10 +192,12 @@ def mask(
 @click.option('--as-mask/--not-as-mask', default=False,
               help="Interpret a band as a mask and output only one class of "
                    "valid data shapes.")
+@click.option('--color', metavar='#rrggbb', help="Feature stroke/fill color")
```

Comment:
@sgillies The help message is kind of ambiguous.  Maybe something like, "Adds a stroke/fill color for each feature in accordance with the GeoJSON simplestyle specification." so users know that the feature is immediately useful in other places and that we're not just arbitrarily modifying the output data.